File: Template.java
/*
 * Author Chaudhary Hamdan
 * Java Template with fast IO Coded for Competitive Programming
 * Github link : https://github.com/hamdan-codes
 */

/*
 * Template available at my Github
 * Repository Link :-
 * https://github.com/hamdan-codes/templates-for-CP/
 */



/*
 * FAST READER is used from www.geeksforgeeks.org
 * Refer at https://www.geeksforgeeks.org/fast-io-in-java-in-competitive-programming/
 */

import java.io.*;
import java.util.*;



/* ***************** Your Classes below ************************ */




















/* ***************** Your Classes Above ************************ */

class Template {             /* ***** Main Class , Rename it as required ***** */

	static Reader br;
	public static void main(String[] args) throws Exception {
		String path = "C:\\Users\\KIIT\\";

		try {
			br = new Reader(path + "input");
		}

		catch (Exception e) {
			br = new Reader();
		}


		try {
			PrintStream stream = new PrintStream(new File(path + "output"));
			System.setOut(stream);
		}

		catch (Exception e) {  }

		StringBuilder sb = new StringBuilder();



		/*
		 * To read variables from keyboard use aliases of functions made below
		 * As for int :
		 * int i = ri();
		 *
		 * To print multiple test cases :
		 * Write as sb.append("Your Answer\n")
		 *
		 * And after end of all testcases write
		 * System.out.println(sb);
		 */

		/* ************* Your Main Function Code Below *************** */



























		/* ************* Your Main Function Code Above *************** */
	}

	/* **************** Your Main Class Functions Below ***************** */












































	/* ****************** Your Main Class Functions Above ****************** */

	/* HELPER FUNCTIONS CREATED BY CHAUDHARY HAMDAN ON 07 February,2021 */

	private static int ri() throws Exception {
		return br.nextInt();
	}
	private static int ri(String s) {
		return Integer.parseInt(s);
	}
	private static String rs() throws Exception {
		return br.readLine();
	}
	private static String[] rss() throws Exception {
		return br.readLine().split(" ");
	}
	private static void pf(Object ob) {
		System.out.print(ob);
	}
	private static void pfn(Object ob) {
		System.out.println(ob);
	}
	private static void pfn(int[] arr) {
		for (int i = 0; i < arr.length; i++) {
			System.out.print(arr[i] + " ");
		}
		System.out.println();
	}

	/* *************************************************************** */
	/*
	 * FAST READER copied from www.geeksforgeeks.org
	 * https://www.geeksforgeeks.org/fast-io-in-java-in-competitive-programming/
	 * */
	static class Reader {
		final private int BUFFER_SIZE = 1 << 16;
		private DataInputStream din;
		private byte[] buffer;
		private int bufferPointer, bytesRead;

		public Reader() {
			din = new DataInputStream(System.in);
			buffer = new byte[BUFFER_SIZE];
			bufferPointer = bytesRead = 0;
		}

		public Reader(String file_name) throws IOException {
			din = new DataInputStream(new FileInputStream(file_name));
			buffer = new byte[BUFFER_SIZE];
			bufferPointer = bytesRead = 0;
		}

		public String readLine() throws IOException {
			byte[] buf = new byte[64]; // line length
			int cnt = 0, c;
			while ((c = read()) != -1) {
				if (c == '\n')
					break;
				buf[cnt++] = (byte) c;
			}
			return new String(buf, 0, cnt);
		}

		public int nextInt() throws IOException {
			int ret = 0;
			byte c = read();
			while (c <= ' ')
				c = read();
			boolean neg = (c == '-');
			if (neg)
				c = read();
			do {
				ret = ret * 10 + c - '0';
			}  while ((c = read()) >= '0' && c <= '9');

			if (neg)
				return -ret;
			return ret;
		}

		public long nextLong() throws IOException {
			long ret = 0;
			byte c = read();
			while (c <= ' ')
				c = read();
			boolean neg = (c == '-');
			if (neg)
				c = read();
			do {
				ret = ret * 10 + c - '0';
			} while ((c = read()) >= '0' && c <= '9');
			if (neg)
				return -ret;
			return ret;
		}

		public double nextDouble() throws IOException {
			double ret = 0, div = 1;
			byte c = read();
			while (c <= ' ')
				c = read();
			boolean neg = (c == '-');
			if (neg)
				c = read();

			do {
				ret = ret * 10 + c - '0';
			} while ((c = read()) >= '0' && c <= '9');

			if (c == '.') {
				while ((c = read()) >= '0' && c <= '9') {
					ret += (c - '0') / (div *= 10);
				}
			}

			if (neg)
				return -ret;
			return ret;
		}

		private void fillBuffer() throws IOException {
			bytesRead = din.read(buffer, bufferPointer = 0, BUFFER_SIZE);
			if (bytesRead == -1)
				buffer[0] = -1;
		}

		private byte read() throws IOException {
			if (bufferPointer == bytesRead)
				fillBuffer();
			return buffer[bufferPointer++];
		}

		public void close() throws IOException {
			if (din == null)
				return;
			din.close();
		}
	}

	/* ************************************************************* */



}

File: Template.cpp
/*
 * Author: Chaudhary Hamdan (https://chaudharyhamdan.me/)
 *
 * Generated at : Sun Apr  3 23:00:20 2022
 *
 * C++ Template for Competitive Programming
 * Github link: https://github.com/hamdan-codes
 *
 * Template available at my Github
 * Repository Link:-
 * https://github.com/hamdan-codes/templates-for-CP/
 */




#include <bits/stdc++.h>

#include <ext/pb_ds/assoc_container.hpp>
#include <ext/pb_ds/exception.hpp>
#include <ext/pb_ds/hash_policy.hpp>
#include <ext/pb_ds/list_update_policy.hpp>
#include <ext/pb_ds/tag_and_trait.hpp>
#include <ext/pb_ds/tree_policy.hpp>
#include <ext/pb_ds/trie_policy.hpp>

#define ff                 first
#define ss                 second
#define int                long long
#define amax(a,b)          if(b>a) swap(a, b)
#define amin(a,b)          if(b<a) swap(a, b)
#define putmax(a,b)        a = max(a, b)
#define putmin(a,b)        a = min(a, b)
#define pb                 push_back
#define mp                 make_pair
#define ii                 int, int
#define pii                pair<ii>
#define vi                 vector<int>
#define vvi                vector<vi>
#define si                 set<int>
#define pi                 print<int>
#define pc                 print<char>
#define ps                 print<string>
#define vpii               vector<pii>
#define vs                 vector<string>
#define vc                 vector<char>
#define gi                 greater<int>
#define umii               unordered_map<ii>
#define mii                map<ii>
#define max_heap           priority_queue<int>
#define min_heap           priority_queue<int, vi, gi>
#define all(x)             x.begin(), x.end()
#define rev(x)             reverse(all(x))
#define setbits(x)         __builtin_popcountll(x)
#define zerobits(x)        __builtin_ctzll(x)
#define deb(x)             cout << #x << "= " << x << endl;
#define py                 cout<<"YES"<<endl
#define pn                 cout<<"NO"<<endl
#define pimp               cout<<"IMPOSSIBLE"<<endl
#define pans               cout<<ans<<endl
#define gcd                __gcd
#define mod                1000000007
#define PI                 3.14159265358979323846264338
#define inf                LONG_LONG_MAX
#define pcs(x,y)           fixed << setprecision(y) << x
#define endl               '\n'
#define maxelem(a)         *max_element(all(a))
#define minelem(a)         *min_element(all(a))
#define lower(s)           transform(all(s), s.begin(), ::tolower)
#define upper(s)           transform(all(s), s.begin(), ::toupper)
#define mk(t,arr,n)        t *arr = new t[n]
#define FI(i,x,y,inc)      for(int i = x; i < y; i += inc)
#define F0(i,x)            FI(i, 0, x, 1)
#define F(i,x,y)           FI(i, x, y, 1)
#define RF(i,x,y)          for(int i = x; i >= y; i--)
#define rep(i,a)           for(auto &i : a)
#define W(x)               int zyx; cin >> zyx; F(x, 1, zyx + 1)
#define W1(x)              F(x, 1, 2)
#define take(a,n)          vi a(n); F0(ari, n) cin >> a[ari]
#define taken(type,a,n)    int n; cin >> n; vector<type> a(n); rep(i, a) cin >> i;
#define din(type,n)        type n; cin >> n
#define in(n)              cin >> n
#define printa(arr,n)      F0(i_ar, n) cout << arr[i_ar] << ' '; cout << endl
#define FAST               ios_base::sync_with_stdio(0); cin.tie(0); cout.tie(0)


using namespace std;

using namespace __gnu_pbds;
using namespace __gnu_cxx;

mt19937                    rng(chrono::steady_clock::now().time_since_epoch().count());
typedef tree<int, null_type, less<int>, rb_tree_tag, tree_order_statistics_node_update> pbds;



void i_o_from_file() {

#ifndef ONLINE_JUDGE
	freopen("C:\\Users\\KIIT\\input", "r", stdin);
	freopen("C:\\Users\\KIIT\\output", "w", stdout);
#endif

}

template<typename T>
void print(int n, ...) {

	va_list ap;
	va_start(ap, n);

	F(i, 1, n + 1) {
		T a = va_arg(ap, T);
		cout << a << ' ';
	}
	cout << endl;

	va_end(ap);

	return;
}




/* ********************* Your Functions and Classes Below ********************** */
















/* ********************* Your Functions and Classes Above ********************** */

void solve_tests() {

	int ans = 0;












	pans;

	return;
}



int32_t main() {

	FAST;

	i_o_from_file();


	/* ******************* Your main function Code Below ******************* */





	W(tc) {


		// cout << "Case #" << tc << ": ";


		solve_tests();





	}




	/* **************** Your main function Code Above ****************** */

	return 0;
}
